server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1409_make_organization_uuid_not_null_on_user_roles.rb

```diff
@@ -0,0 +1,29 @@
+#
+# SonarQube, open source software quality management tool.
+# Copyright (C) 2008-2014 SonarSource
+# mailto:contact AT sonarsource DOT com
+#
+# SonarQube is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# SonarQube is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+#
+# SonarQube 6.2
+#
+class MakeOrganizationUuidNotNullOnUserRoles < ActiveRecord::Migration
+
+  def self.up
+    execute_java_migration('org.sonar.db.version.v62.MakeOrganizationUuidNotNullOnUserRoles')
```

Comment:
I'm surprised content of this commit is not part of commit "SONAR-8245 populate user_roles.organization_uuid during upgrade". After all, the changes to DB unit tests in "SONAR-8245 populate user_roles.organization_uuid during upgrade" are only required because the column is made not nullable.

I think those two commits should be sqashed together.